CondFormats/GEMObjects/interface/GEMELMap.h

```diff
@@ -51,6 +51,7 @@ class GEMELMap {
   
  public:
   // size of ID bits
+  static const int vfatTypeV3_ = 11;     // ID size from VFat
```

Comment:
Here you are adding static data members. Normally, they are not expected to be stored when you serialise the object.  Is it the case? Did you test that you can correctly read old payloads with the new class, and new payloads with the class version pre-change?

Reply:
@ggovi - I've tested with the old payloads and these have no differences.

Reply:
As far as I understood after the discussion during the last reco meeting, this will be eventually picked from the DB.

Reply:
Ok, if you could also test that the previous code version can read new payload, that should complete the compatibility...

Reply:
@perrotta - for this to be picked up from DB, we will need to add in a new variable to DB, which means we need to make a new class GEMELMap. We didn't want to make a new class, so we were trying to make do with the current variables in DB. 
Shall we go ahead and make a new DB and new mapping class for it?